src/coreclr/tools/Common/JitInterface/ThunkGenerator/ThunkInput.txt

```diff
@@ -315,7 +315,7 @@ FUNCTIONS
     bool getStaticFieldContent(CORINFO_FIELD_HANDLE field, uint8_t *buffer, int bufferSize, int valueOffset, bool ignoreMovableObjects);
     bool getObjectContent(CORINFO_OBJECT_HANDLE obj, uint8_t *buffer, int bufferSize, int valueOffset);
     CORINFO_CLASS_HANDLE getStaticFieldCurrentClass(CORINFO_FIELD_HANDLE    field,  BoolStar pIsSpeculative);
-    CORINFO_VARARGS_HANDLE getVarArgsHandle(CORINFO_SIG_INFO       *pSig, void                  **ppIndirection);
+    CORINFO_VARARGS_HANDLE getVarArgsHandle(CORINFO_SIG_INFO       *pSig, CORINFO_METHOD_HANDLE   methHnd, void                  **ppIndirection);
     bool canGetVarArgsHandle(CORINFO_SIG_INFO       *pSig);
```

Comment:
`canGetVarArgsHandle` method can be deleted. It is useless - left-over from fragile NGen.

Reply:
Same for `canGetCookieForPInvokeCalliSig`. It can be deleted as well.

(You may want to submit the JIT/EE interface edits as separate prep-work PR.)